firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -2405,6 +2416,7 @@ extension BrowserViewController: SearchViewControllerDelegate {
                     TelemetryWrapper.EventValue.fxSuggestionTelemetryInfo.rawValue: telemetryInfo,
                     TelemetryWrapper.EventValue.fxSuggestionPosition.rawValue: position,
                     TelemetryWrapper.EventValue.fxSuggestionDidTap.rawValue: didTap,
+                    TelemetryWrapper.EventValue.fxSuggestionDidAbandonEngagement.rawValue: didAbandonEngagement,
```

Comment:
`didAbandonSearchSession`? `isAbandonment`? Some other, better name to tie this to the engagement and abandonment events?

Reply:
`didAdandonSearchSession` sounds descriptively good to me!

Reply:
Great! I pushed a second commit renaming `SearchEngagementState` to `SearchSessionState`, to be more consistent with this.